Feature/ot111 22

denisdimarco opened the pull request:
[https://alkemy-labs.atlassian.net/browse/OT111-22](https://alkemy-labs.atlassian.net/browse/OT111-22)
- Members Entity
- Members Script
- Members Repository

denisdimarco committed:
OT111-22 - Added JPA & Postgresql dependencies to build.gradle

denisdimarco committed:
OT111-22 - Added sql script to create members table

denisdimarco committed:
OT111-22 - MemberEntity Created

denisdimarco committed:
OT111-22 - Updated member.sql

denisdimarco committed:
OT111-22 - Change id data type in Entity & script

denisdimarco committed:
OT111-22 - MemberRepository created.

OsvaldoGWP commented on scripts/v0.1__members.sql:
```diff
@@ -0,0 +1,19 @@
+create table if not EXISTS members (
```
nombres de las tablas en singular por la convencion que planteamos en un inicio que esta en la wiki

OsvaldoGWP commented on scripts/v0.1__members.sql:
```diff
@@ -0,0 +1,19 @@
+create table if not EXISTS members (
+--  Fields
+    id int8 NOT NULL GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR NOT NULL,
```
añadir longitud

denisdimarco committed:
test comit

denisdimarco committed:
Merge branch 'testbranch' into develop

OsvaldoGWP commented on src/main/java/com/alkemy/ong/model/entity/MemberEntity.java:
```diff
@@ -0,0 +1,48 @@
+@Setter
+@Entity
+@Table(name = "members")
+@SQLDelete(sql = "UPDATE members SET deleted = true WHERE id=?")
```
recordar de actualizar esto ya q la tabla ahora esta en singular

OsvaldoGWP commented on src/main/java/com/alkemy/ong/repository/MemberRepository.java:
```diff
@@ -0,0 +1,9 @@
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface MemberRepository extends JpaRepository <MemberEntity, Integer> {
```
el Id es Long cambiar integer por long, JpaRepository <MemberEntity, Long>

denisdimarco committed:
OT111-22 - PR #6 resolved

v0.1__members.sql:
- Change table name to 'member
- Added VARCHAR lengths 

MemberEntity.java:
- Change table name
- Added @NotNull validation to fields

MemberRepository:
-Fixed Id type to Long 

Added dependency for hibernate-validator

Updated application.yml

denisdimarco committed:
Revert "Merge branch 'develop' into feature/OT111-22"

This reverts commit f17907c598c95f996ae42baca4b60d986017c64f.

denisdimarco committed:
Merge branch 'develop' into feature/OT111-22

denisdimarco committed:
Changed member_pk name

denisdimarco committed:
Added Spring Web Dependency

denisdimarco committed:
Update v0.1__member.sql

denisdimarco committed:
Update MemberEntity.java

OsvaldoGWP commented on build.gradle:
```diff
@@ -20,9 +20,19 @@ repositories {
+	runtimeOnly 'org.postgresql:postgresql'
 	annotationProcessor 'org.projectlombok:lombok'
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
+	testImplementation 'org.springframework.boot:spring-boot-starter-test'
```
esto esta repetido con la linea 33
